### ISSUE TITLE
Training a fine-tuned AI model Problem

### ISSUE THREAD

**Issue by tsapap** [charonn0/Xojo-OpenAI#7]
Hello, why do I always receive this error when using the Training a fine-tuned AI model method from
https://github.com/charonn0/Xojo-OpenAI/wiki/Examples#training-a-fine-tuned-ai-model
as shown in the attached image?

![image](https://github.com/user-attachments/assets/6a9b6dcd-f7a1-46fd-8b57-6a8fed7fd73d)

Shared Function Create(TrainingFile As FolderItem, Overwrite As Boolean = False) As OpenAI.FineTuneData
  Dim bs As BinaryStream = BinaryStream.Create(TrainingFile, Overwrite)
  bs.Close
  Return New FineTuneData(TrainingFile)
End Function

I am trying to train ChatGPT with my own information to customize the responses based on my documents.

**Comment by charonn0** [comment]
An IOException on `BinaryStream.Create(TrainingFile, Overwrite)` where `Overwrite=False` suggests that the specified file already exists. 

The example code uses `FineTuneData.Create()`to create a new, blank file. To load an existing file, modify the example code to use `FineTuneData.Load()`: 

```realbasic
 Dim trainer As OpenAI.FineTuneData = OpenAI.FineTuneData.Load(datafile) 
```
